test/twofishes/client_test.rb

```diff
@@ -29,6 +29,90 @@ def self.geocode(location)
       assert result.parents
     end
 
+    it 'should accept max keyword argument' do
+      Twofishes::Client.expects(:thrift_client).returns(ZurichWithParents)
+      result = Twofishes::Client.geocode('Zurich', max: 5).first
+      assert result.parents
+    end
+
+    it 'should accept ll keyword argument' do
+      Twofishes::Client.expects(:thrift_client).returns(ZurichWithParents)
+      result = Twofishes::Client.geocode('Zurich', ll: "47,8").first
+      assert result.displayName
```

Comment:
For consistencies sake, I would do the same assertion as in the first test, see https://github.com/arepalabs/twofishes-ruby/blob/include-all-geocode-parameters/test/twofishes/client_test.rb#L23